OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -119,14 +114,7 @@ protected virtual void DamageStateChanged(Actor self)
 				DefaultAnimation.ReplaceAnim(NormalizeSequence(self, DefaultAnimation.CurrentSequence.Name));
 		}
 
-		protected virtual void Tick(Actor self)
-		{
-			if (Info.AimSequence == null)
-				return;
-
-			var sequence = Attack.IsAttacking ? Info.AimSequence : Info.Sequence;
-			DefaultAnimation.ReplaceAnim(sequence);
-		}
+		protected virtual void Tick(Actor self) { }
```

Comment:
Better to remove `ITick` completely, and if a subclass wants to tick then they can implement the interface themselves.